waspc/data/Generator/templates/react-app/src/test/vitest/helpers.tsx

```diff
@@ -65,3 +67,9 @@ export function mockServer(): {
 
   return { server, mockQuery }
 }
+
+type InternalQuery<Input, Output> = {
```

Comment:
We are hiding the `route` from user?

Reply:
Yes, for the time being, reasoning here: https://github.com/wasp-lang/wasp/pull/1055#discussion_r1150429257 (spoiler: not much of a reasoning)